tshock/bootstrap.sh

```diff
@@ -17,20 +17,25 @@ if [ $STORAGETYPE = "mysql" ]; then
 fi
 
 WORLD_PATH="/root/.local/share/Terraria/Worlds/$WORLD_FILENAME"
+if [ ! -f "/config/serverconfig.txt" ]; then
```

Comment:
Thinking about my comment about the `/config` volume, I think we should place the `serverconfig.txt` in the `/root/.local/share/Terraria/Worlds/` volume.